src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceServiceSpec.scala

```diff
@@ -41,6 +41,10 @@ class WorkspaceServiceSpec extends FreeSpec with ScalaFutures with ScalatestRout
       MockWorkspaceServer.mockValidWorkspace.namespace.get,
       MockWorkspaceServer.mockValidWorkspace.name.get)
 
+    val attributeUpdatePath = ApiPrefix + "/%s/%s/updateAttributes".format(
+      MockWorkspaceServer.mockValidWorkspace.namespace.get,
+      MockWorkspaceServer.mockValidWorkspace.name.get)
```

Comment:
Here's where you'd add an invalid attributeUpdatePath using mockInvalidWorkspace.